### DISCUSSION TITLE
How to run stuff that i have in my .xinitrc in wayfire?

### DISCUSSION THREAD

**Discussion by LinArcX** [WayfireWM/wayfire#1541] [Q&A] [answered]
Hello, this is part of my .xinitrc file and I use it to run dwm:
```
numlockx on &
dunst &
alacritty &
/home/linarcx/brave/brave-browser &
emacsclient -s /var/run/user/1000/emacs/server -c -n -a emacs &

while true; do
 inotifywait ~/.config/BraveSoftware/Brave-Browser/Default/Bookmarks >/dev/null 2>&1; cp /home/linarcx/.config/BraveSoftware/Brave-Browser/Default/Bookmarks /home/linarcx/VoidConf/home/.config/Bookmarks
done &

```

I want to run these applications and scripts at wayfire startup. I looked at the [autostart] section in the config file, but to be honest, don't have any idea how to run my applications through it.

**Comment by soreau** [comment, marked as answer]
For numlock default state, there is a setting [here](https://github.com/WayfireWM/wayfire/wiki/Configuration#inputkb_numlock_default_state). For [autostart] plugin, the syntax is `anything = command` where anything can be anything unique to the rest and command is the command passed to sh.

**Reply by LinArcX** [reply]
Thank you. The second part works like a charm. Though the link you provided isn't valid.

**Comment by soreau** [comment]
```
input.kb_numlock_default_state

Default numlock state when Wayfire starts. The default is false.
```
